.github/workflows/post-atlas-operator-release.yaml

```
@@ -0,0 +1,102 @@
+name: Create PR with Atlas Operator Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: true
+
+jobs:
+  create-pr:
+    permissions: write-all
+    name: Create Release branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repositories
+        run: |
+          mkdir operator
+          cd operator
+          git clone https://github.com/mongodb/mongodb-atlas-kubernetes.git
+          cd ..
+          mkdir helm
+          cd helm
+          git clone https://github.com/Sugar-pack/helm-charts.git
+          cd ..
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Copy crd template files
+        run: |
+          cp -a operator/mongodb-atlas-kubernetes/bundle/manifests/atlas.mongodb.com_atlasbackuppolicies.yaml helm/helm-charts/charts/atlas-operator-crds/templates
```

I'd copy the entire directory, in case someone adds new CRD
```suggestion
          cp -r operator/mongodb-atlas-kubernetes/bundle/manifests/ helm/helm-charts/charts/atlas-operator-crds/
```